scripts/team.js

```diff
@@ -1,189 +1,261 @@
-"use strict";
+'use strict';
+
+const favClassesDataSet = {
+	linkedIn: 'fa fa-linkedin',
+	twitter: 'fa fa-twitter',
+	github: 'fa fa-github',
+	facebook: 'fa fa-facebook'
+};
+
+const mapProfileLinksToFavClasses = profileName => {
+	return favClassesDataSet[profileName];
+};
 
 const teamData = [
 	{
 		name: 'Smaranjit Ghose',
 		position: 'The Board',
-		linkedIn: 'https://www.linkedin.com/in/smaranjitghose/',
-		github: 'https://github.com/smaranjitghose',
-		twitter:'',
-		image: 'https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80'
+		image: 'smaranjit_ghose.jpg',
+		profiles: [
+			{
+				linkedIn: 'https://www.linkedin.com/in/smaranjitghose/',
+				github: 'https://github.com/smaranjitghose',
+				twitter: '#!'
+			}
+		]
 	},
 
 	{
 		name: 'Anush Bhatia',
 		position: 'The Board',
-		linkedIn: 'https://www.linkedin.com/in/anush-bhatia-aa500a158/',
-		github: 'https://github.com/anushbhatia',
-		twitter: '',
-		image: ''
+		image: 'anush_bhatia.jpg',
+		profiles: [
+			{
+				linkedIn: 'https://www.linkedin.com/in/anush-bhatia-aa500a158/',
+				github: 'https://github.com/anushbhatia',
+				twitter: '#!'
+			}
+		]
 	},
 
 	{
 		name: 'Saswat Nayak',
 		position: 'The Board',
-		linkedIn: 'https://www.linkedin.com/in/saswat-nayak-a41212166/',
-		github: 'https://github.com/swat1998',
-		cssClass: 'board',
-		image: 'assets/Images/team/saswat_nayak.jpg'
+		image: 'saswat_nayak.jpg',
```

Comment:
Can you please to me explain how are we fetching the image here without giving the path of it in the assets folder?

Reply:
Since the pathname is the same for all of them except for the filename , it is loaded directly on  number  245 and 246 Sir.